colour/models/rgb/transfer_functions/log.py

```diff
@@ -0,0 +1,142 @@
+#
+"""
+Log2 Shaper Implementation
+==========================
+
+Defines the *Log2 shaper* log encodings:
+
+-   :func:`colour.models.log_encoding_Log2`
+-   :func:`colour.models.log_decoding_Log2`
+
+References
+----------
+-   :cite:`TheAcademyofMotionPictureArtsandSciencesa` :
+    The Academy of Motion Picture Arts and Sciences,
+    Science and Technology Council,
+    & Academy Color Encoding System (ACES) Project Subcommittee.(n.d.-a).
+    ACESutil.Lin_to_Log2_param.ctl. Retrieved June 14, 2020,
+    from https://github.com/ampas/aces-dev/blob/\
+518c27f577e99cdecfddf2ebcfaa53444b1f9343/transforms/ctl/utilities/ACESutil.Lin_to_Log2_param.ctl
+-   :cite:`TheAcademyofMotionPictureArtsandSciencesb` :
+    The Academy of Motion Picture Arts and Sciences,
+    Science and Technology Council,
+    & Academy Color Encoding System (ACES) Project Subcommittee.(n.d.-b).
+    ACESutil.Log2_to_Lin_param.ctl. Retrieved June 14, 2020,
+    from https://github.com/ampas/aces-dev/blob/\
+518c27f577e99cdecfddf2ebcfaa53444b1f9343/transforms/ctl/utilities/ACESutil.Log2_to_Lin_param.ctl
+"""
+
+from __future__ import division, unicode_literals
+
+import numpy as np
+from colour.utilities import from_range_1, to_domain_1
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013-2020 - Colour Developers'
+__license__ = 'New BSD License - https://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-developers@colour-science.org'
+__status__ = 'Production'
+
+__all__ = ['log_encoding_Log2', 'log_decoding_Log2']
+
+
+def log_encoding_Log2(lin,
+                      middle_grey=0.18,
+                      min_exposure=0.18 * 2 ** -6.5,
+                      max_exposure=0.18 * 2 ** 6.5):
+    """
+    Defines the *Log2 shaper* log encoding function.
+
+    Parameters
+    ----------
+    lin : numeric or array_like
+          Linear data to undergo encoding.
+    middle_grey : numeric, optional
+                  'Middle Grey' exposure value.
+    min_exposure : numeric, optional
+                   Minimum exposure level.
+    max_exposure : numeric, optional
+                   Maximum exposure level.
+
+    Returns
+    -------
+    numeric or ndarray
+        Non-linear *Log2 shaper* encoded data
+
+    Notes
```

Comment:
If the section is empty, we should get rid of it, maybe here is a good place to mention how this function is used to build various shapers in the ACES OCIO configuration.

Reply:
Sounds good, will go about it!